files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -96,7 +96,7 @@
     action: keep
```

Comment:
I know we have service monitors on MCs -- do those also get us the same cadvisor metrics? Maybe we should add this to the MC configuration as well

Reply:
Yes, I think you should add it to the MC as well.

Reply:
Sure, will do that.

Reply:
@hervenicol does adding the namespace here result in _everything_ in that namespace getting scraped automatically? We have some resources in this namespace in WCs that we don't want to scrape

Reply:
Only cadvisor data. But all pods in the namespace will be scraped.
If you know how to determine what you don't want to scrape, you could add a `drop` action.

Like:
```
  - source_labels: [container]
    regex: .*IWannaDrop.*
    action: drop
```
Just make sure it does not catch too many metrics!

The current config is a bit hard to read and maintain, hopefully when we switch to serviceMonitors it's easier.